app/javascript/readingList/readingList.jsx

```diff
@@ -130,8 +130,21 @@ export class ReadingList extends Component {
             value="Your reading list is empty"
           />
           <p class="color-base-60 pt-2">
-            Hit the <span class="fw-bold">Save</span> button to start your
-            Collection.
+            Click the{' '}
+            <span class="fw-bold">
+              bookmark reaction
+              <svg
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                className="crayons-icon"
```

Comment:
```suggestion
                className="crayons-icon mx-1"
```

just for extra spacing on left or right.
no blocker, take it or leave it :)

Reply:
Thanks @ludwiczakpawel!